static/src/javascripts/projects/common/utils/drop-while.js

```diff
@@ -0,0 +1,17 @@
+define(function () {
```

Comment:
Aye, like [Lodash utils are](https://github.com/guardian/frontend/blob/master/static/src/javascripts/components/lodash-amd/objects/assign.js) - which is then used for their docs. If we're going to start using utils like this a lot, it'll be useful if they're all documented like this (though I know most aren't so far...)